cartridges/custom_cartridge_1/cartridge/controllers/Account.js

```diff
@@ -0,0 +1,188 @@
+'use strict';
+
+/**
+ * @namespace Account
+ */
+
+var server = require('server');
+server.extend(module.superModule);
+
+var csrfProtection = require('*/cartridge/scripts/middleware/csrf');
+var userLoggedIn = require('*/cartridge/scripts/middleware/userLoggedIn');
+var consentTracking = require('*/cartridge/scripts/middleware/consentTracking');
+
+/**
+ * Checks if the email value entered is correct format
+ * @param {string} email - email string to check if valid
+ * @returns {boolean} Whether email is valid
+ */
+function validateEmail(email) {
+    var regex = /^[\w.%+-]+@[\w.-]+\.[\w]{2,6}$/;
+    return regex.test(email);
+}
+
+/**
+ * Account-EditProfile : The Account-EditProfile endpoint renders the page that allows a shopper to edit their profile. The edit profile form is prefilled with the shopper's first name, last name, phone number and email
+ * @name Base/Account-EditProfile
+ * @function
+ * @memberof Account
+ * @param {middleware} - server.middleware.https
+ * @param {middleware} - csrfProtection.generateToken
+ * @param {middleware} - userLoggedIn.validateLoggedIn
+ * @param {middleware} - consentTracking.consent
+ * @param {category} - sensitive
+ * @param {renders} - isml
+ * @param {serverfunction} - get
+ */
+server.append(
+    'EditProfile',
+    server.middleware.https,
+    csrfProtection.generateToken,
+    userLoggedIn.validateLoggedIn,
+    consentTracking.consent,
+    function (req, res, next) {
+        var accountHelpers = require('*/cartridge/scripts/account/accountHelpers');
+        var accountModel = accountHelpers.getAccountModel(req);
+        var profileForm = server.forms.getForm('profile');
```

Comment:
Do you really need to get the form from the server? Is there anywhere else, where you can find it?

Reply:
No, i don`t. I can get the profileForm  from here:  var profileData = res.getViewData();